src/Parallel/Helper.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "utils/env.h"
+
+namespace seissol {
+template <typename T>
+void printCommThreadInfo(const T& mpiBasic) {
+  bool useThread = utils::Env::get<bool>("SEISSOL_COMMTHREAD", true);
+  if (mpiBasic.isSingleProcess()) {
+    logInfo(mpiBasic.rank()) << "Using polling for advancing MPI communication, due to having only "
+                                "one MPI rank running.";
+  }
+  if (useThread) {
+    logInfo(mpiBasic.rank()) << "Using a communication thread for advancing MPI communication.";
```

Comment:
so if SEISSOL_COMMTHREAD=True and one node, we will get:
```
Using polling for advancing MPI communication, due to having only one MPI rank running.
Using a communication thread for advancing MPI communication.
```
Which is not what expected.